src/SDKs/Automation/Automation.Tests/ScenarioTests/AutomationTests.cs

```diff
@@ -467,7 +467,7 @@ public void CanCreateSourceControlSyncJob()
                     Assert.Equal(retrievedSyncJob.ProvisioningState, syncJob.ProvisioningState);
                     Assert.Equal(retrievedSyncJob.StartTime, syncJob.StartTime);
                     Assert.Equal(retrievedSyncJob.EndTime, syncJob.EndTime);
-                    Assert.Equal(retrievedSyncJob.StartType, syncJob.StartType);
+                    //Assert.Equal(retrievedSyncJob.SyncType, syncJob.SyncType);
 
```

Comment:
Why is this commented

Reply:
The web service changes have not reached the region that is used to record the tests. If you prefer, I can skip the test and rerecord it for the next SDK update. Let me know. Thanks.

Reply:
Fixed.